### ISSUE TITLE
Armed Info do not work corectlly...

### ISSUE THREAD

**Issue by DejanBukovec** [petrleocompel/hikaxpro_hacs#12]
Armed Info do not work corectlly...

In "async_setup_entry" Im change:
`_LOGGER.debug("Adding device: %s", zone)`

to:
`_LOGGER.error("Adding device: %s", zone)`

To get all device informations into log:
```
2023-02-28 15:36:25.590 ERROR (MainThread) [custom_components.hikvision_axpro.sensor] Adding device: ZoneList(zone=Zone(id=0, name='Kurilno Olje - Magnet', status=<Status.ONLINE: 'online'>, tamper_evident=False, shielded=False, bypassed=False, armed=False, is_arming=False, alarm=False, sub_system_no=1, linkage_sub_system=[1], detector_type=<DetectorType.MAGNETIC_CONTACT: 'magneticContact'>, stay_away=False, zone_type=<ZoneType.INSTANT: 'Instant'>, zone_attrib=<ZoneAttrib.WIRELESS: 'wireless'>, device_no=2, abnormal_or_not=False, charge='normal', charge_value=100, signal=121, temperature=5, humidity=None, model='0x00006', is_via_repeater=False, version='V1.0.25', magnet_open_status=False, input_list=[InputList(id=1, enabled=False, mode='normalClose'), InputList(id=2, enabled=False, mode='normalClose')], is_support_add_type=True, access_module_type=None, module_channel=None))
2023-02-28 15:36:25.594 ERROR (MainThread) [custom_components.hikvision_axpro.sensor] Adding device: ZoneList(zone=Zone(id=1, name='Pisarna', status=<Status.ONLINE: 'online'>, tamper_evident=False, shielded=False, bypassed=False, armed=False, is_arming=False, alarm=False, sub_system_no=2, linkage_sub_system=[2], detector_type=<DetectorType.WIRELESS_PIR_CAM_DETECTOR: 'pircam'>, stay_away=False, zone_type=<ZoneType.INSTANT: 'Instant'>, zone_attrib=<ZoneAttrib.WIRELESS: 'wireless'>, device_no=4, abnormal_or_not=False, charge='normal', charge_value=100, signal=121, temperature=24, humidity=None, model='0x00012', is_via_repeater=False, version='V1.2.0', magnet_open_status=None, input_list=None, is_support_add_type=None, access_module_type=None, module_channel=None))
2023-02-28 15:36:25.595 ERROR (MainThread) [custom_components.hikvision_axpro.sensor] Adding device: ZoneList(zone=Zone(id=2, name='Vhod', status=<Status.ONLINE: 'online'>, tamper_evident=False, shielded=False, bypassed=False, armed=False, is_arming=False, alarm=False, sub_system_no=1, linkage_sub_system=[1], detector_type=<DetectorType.PASSIVE_INFRARED_DETECTOR: 'passiveInfraredDetector'>, stay_away=False, zone_type=<ZoneType.DELAY: 'Delay'>, zone_attrib=<ZoneAttrib.WIRELESS: 'wireless'>, device_no=5, abnormal_or_not=False, charge='normal', charge_value=100, signal=121, temperature=16, humidity=None, model='0x00001', is_via_repeater=False, version='V1.0.25', magnet_open_status=None, input_list=None, is_support_add_type=None, access_module_type=None, module_channel=None))
2023-02-28 15:36:25.596 ERROR (MainThread) [custom_components.hikvision_axpro.sensor] Adding device: ZoneList(zone=Zone(id=3, name='Otroška Soba', status=<Status.ONLINE: 'online'>, tamper_evident=False, shielded=False, bypassed=False, armed=False, is_arming=False, alarm=False, sub_system_no=2, linkage_sub_system=[2], detector_type=<DetectorType.PASSIVE_INFRARED_DETECTOR: 'passiveInfraredDetector'>, stay_away=False, zone_type=<ZoneType.INSTANT: 'Instant'>, zone_attrib=<ZoneAttrib.WIRELESS: 'wireless'>, device_no=6, abnormal_or_not=False, charge='normal', charge_value=100, signal=121, temperature=21, humidity=None, model='0x00001', is_via_repeater=False, version='V1.0.25', magnet_open_status=None, input_list=None, is_support_add_type=None, access_module_type=None, module_channel=None))
2023-02-28 15:36:25.598 ERROR (MainThread) [custom_components.hikvision_axpro.sensor] Adding device: ZoneList(zone=Zone(id=4, name='Dnevna soba', status=<Status.ONLINE: 'online'>, tamper_evident=False, shielded=False, bypassed=False, armed=False, is_arming=False, alarm=False, sub_system_no=1, linkage_sub_system=[1], detector_type=<DetectorType.PASSIVE_INFRARED_DETECTOR: 'passiveInfraredDetector'>, stay_away=False, zone_type=<ZoneType.INSTANT: 'Instant'>, zone_attrib=<ZoneAttrib.WIRELESS: 'wireless'>, device_no=7, abnormal_or_not=False, charge='normal', charge_value=100, signal=121, temperature=25, humidity=None, model='0x00001', is_via_repeater=False, version='V1.0.25', magnet_open_status=None, input_list=None, is_support_add_type=None, access_module_type=None, module_channel=None))
2023-02-28 15:36:25.599 ERROR (MainThread) [custom_components.hikvision_axpro.sensor] Adding device: ZoneList(zone=Zone(id=5, name='Garaža', status=<Status.ONLINE: 'online'>, tamper_evident=False, shielded=False, bypassed=False, armed=True, is_arming=False, alarm=False, sub_system_no=4, linkage_sub_system=[4], detector_type=<DetectorType.PASSIVE_INFRARED_DETECTOR: 'passiveInfraredDetector'>, stay_away=False, zone_type=<ZoneType.INSTANT: 'Instant'>, zone_attrib=<ZoneAttrib.WIRELESS: 'wireless'>, device_no=9, abnormal_or_not=False, charge='normal', charge_value=90, signal=121, temperature=4, humidity=None, model='0x00001', is_via_repeater=False, version='V1.0.25', magnet_open_status=None, input_list=None, is_support_add_type=None, access_module_type=None, module_channel=None))

```
Zone with ID 5 belong to "Garage" which is currently armed. In log I see armed=True but in GUI it show Off:
![slika](https://user-images.githubusercontent.com/52787044/221860427-2494a6ec-ffa2-4198-b018-6d9443e9f0d5.png)

It create "sensor.ax_pro_armed_5" but is all time Off same as for other sensors which are currently not armed. Any idea why this happen? For other new entities I can't currently report if work correctly(I need to test them)...

**Comment by DejanBukovec** [comment]
Hi,
Is this problem only with Armed state or all other will have similar problems(Because I copy code)?

**Comment by petrleocompel** [comment]
You can see the commit. Comparison was made "Ture == Status.Online". I will check other parts. It was just simple mistake (cherry picked code from you)
For now it was only armed state.

**Comment by DejanBukovec** [comment]
I see this but asking because all use same code... Im not very familiar with Python(I know C#, Delphi, Java, C ...) but does not need to be something like:
```
    @property
    def is_on(self) -> bool | None:
        """Return true if the binary sensor is on."""
        if self.coordinator.zones and self.coordinator.zones[self.zone.id]:
            value = self.coordinator.zones[self.zone.id].armed
            return value
        else:
            return False
```

But maybe Python work different as other languages. My logic will tell me that if "if" is true then it return what is armed(True or False) but if "if" is not true then go to else and it return False...


This ONLINE "issue" is in:
HikWirelessExtMagnetDetector
HikMagneticContactDetector
HikTamperDetection
HikBypassDetection
HikAlarmInfo
HikStayAwayInfo
HikIsViaRepeaterInfo

**Comment by petrleocompel** [comment]
Yes you are right I did not check them all yet.

**Comment by petrleocompel** [comment]
I see now more things to fix. Be patient.

**Comment by petrleocompel** [comment]
This should be fixed now

**Comment by petrleocompel** [comment]
Fixed in 0.6.2
